slim_trees/lgbm_booster.py

```diff
@@ -198,8 +198,9 @@ def _compress_booster_handle(model_string: str) -> Tuple[str, List[dict], str]:
     return front_str, trees, back_str
 
 
-def _validate_tree_structure(tree: dict) -> bool:
```

Comment:
No idea how this slipped under our radar 😅 let's hope this doesn't break any tests